### ISSUE TITLE
mockery-0.3.0 test suite failure

### ISSUE THREAD

**Issue by peti** [hspec/mockery#6]
Citing from http://hydra.cryp.to/build/995352/nixlog/9/raw:

```
Running 1 test suites...
Test suite spec: RUNNING...

Test.Mockery.Directory
  withFile
    creates a temporary file with the given contents
  touch
    creates an empty file
    creates any missing directories
    when file already exists
      updates modification time FAILED [1]
      does not modify file content
Test.Mockery.Logging
  captureLogs
    returns all log messages of an action
    restores the original log sink

Failures:

  test/Test/Mockery/DirectorySpec.hs:36: (best-effort)
  1) Test.Mockery.Directory.touch, when file already exists, updates modification time
       predicate failed on: 2015-07-23 21:54:37 UTC

Source locations marked with "best-effort" are calculated heuristically and may be incorrect.

Randomized with seed 1829322960

Finished in 0.1320 seconds
7 examples, 1 failure
Test suite spec: FAIL
```

**Comment by aycanirican** [comment]
@sol why not increase the delay? Since getModificationTime doesn't have sub-second resolution.

https://github.com/hspec/mockery/blob/master/test/Test/Mockery/DirectorySpec.hs#L38

**Comment by sol** [comment]
I don't know in which environments we have the sub-second resolution. I've it on my box, but on Travis we don't.

@peti I guess it's possible to make the tests pass by setting `TRAVIS=true`.

**Comment by peti** [comment]
I've added https://github.com/peti/nixpkgs/commit/bea689d6dad5d2a278c8e4420b0cb58156e578d4. Hopefully that solves the issue for the time being.

**Comment by peti** [comment]
This error appears to be new:

```
Running 1 test suites...
Test suite spec: RUNNING...

Test.Mockery.Directory
  withFile
    creates a temporary file with the given contents
  touch
    creates an empty file
    creates any missing directories
    when file already exists
      updates modification time
      does not modify file content
Test.Mockery.Logging
  captureLogs
    returns all log messages of an action
    restores the original log sink FAILED [1]

Failures:

  test/Test/Mockery/LoggingSpec.hs:24:
  1) Test.Mockery.Logging.captureLogs restores the original log sink
       test/Test/Mockery/LoggingSpec.hs:29:
       expected: [LogRecord {logRecordLevel = TRACE, logRecordLocation = Nothing, logRecordMessage = "this should not be captured"}]
        but got: [LogRecord {logRecordLevel = TRACE, logRecordLocation = Just (Location {locationPackage = "main", locationModule = "Test.Mockery.LoggingSpec", locationFile = "test/Test/Mockery/LoggingSpec.hs", locationLine = 28, locationColumn = 5}), logRecordMessage = "this should not be captured"}]

Randomized with seed 49684198

Finished in 1.0131 seconds
7 examples, 1 failure
Test suite spec: FAIL
```

Full log is at http://hydra.cryp.to/build/1054498/nixlog/1/raw.

**Comment by sol** [comment]
This is due to GHC 7.10.2 source locations.  Fixed in 7a7ecf1bf0e7e7888db3d1769613a3789e36b314.
